wandb/sdk/artifacts/artifact.py

```diff
@@ -1086,53 +1086,28 @@ def __setitem__(self, name: str, item: data_types.WBValue) -> ArtifactManifestEn
             The added manifest entry
 
         Raises:
-            ArtifactFinalizedError: if the artifact has already been finalized.
-
-        Examples:
-            Basic usage:
-            ```python
-            artifact = wandb.Artifact("my_table", type="dataset")
-            table = wandb.Table(
-                columns=["a", "b", "c"], data=[(i, i * 2, 2**i) for i in range(10)]
-            )
-            artifact["my_table"] = table
-
-            wandb.log_artifact(artifact)
-            ```
-
-            Retrieving an object:
-            ```python
-            artifact = wandb.use_artifact("my_table:latest")
-            table = artifact["my_table"]
-            ```
+            ArtifactFinalizedError: If the artifact is already finalized.
```

Comment:
```suggestion
            ArtifactFinalizedError: You cannot make changes to the current artifact version because it is finalized. Log a new artifact version instead.
```